content/posts/interview_template_AndersNord_2023-02-05.md

```diff
@@ -0,0 +1,54 @@
+title: Intervju med medlem: Anders Nord
+date: 2023-02-05
+author: Anders Nord
+category: Intervjuer
+tags: intervju, klimat, ifk, medlemmar
```

Comment:
```suggestion
tags: intervju, klimat, ifk, medlemmar
data: 
```

Sorry, need to add this back we the image is to be fetched.

Reply:
Yup. Fixed in commits below.